pages/common/crictl.md

```diff
@@ -0,0 +1,36 @@
+# crictl
+
+> Command-line for CRI-compatible container runtimes.
+> More information: <https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md>.
+
+- List all kubernetes pods (Ready and NotReady):
+
+`crictl pods`
+
+- List all containers (Running and Exited):
+
+`crictl ps --all`
+
+- List all images:
+
+`crictl images`
+
+- Display information about one or more containers:
+
+`crictl inspect {{container_id}}`
```

Comment:
If you mentioned `one or more container` in the description, is it better to change to this:

```suggestion
`crictl inspect {{container_id}} {{container_id}}`
```